pico_project.py

```diff
@@ -894,7 +894,13 @@ def GenerateCMake(folder, params):
                 )
 
     cmake_header2 = ("# Pull in Raspberry Pi Pico SDK (must be before project)\n"
-                "include(pico_sdk_import.cmake)\n\n"
+                "include(pico_sdk_import.cmake)\n"
+                "find_program(ELF2UF2_BIN ELF2UF2)\n"
```

Comment:
I think any change like this ought to be made in `pico_sdk_import.cmake` (which comes from https://github.com/raspberrypi/pico-sdk/blob/master/external/pico_sdk_import.cmake ) or similar, rather than only being made "locally"?